src/mlj_model_interface.jl

```diff
@@ -53,74 +55,61 @@ end
 
 
 # # FIT AND  UPDATE
+include("fit_utils.jl")
+include("entity_embedding_utils.jl")
 
-const ERR_BUILDER =
-    "Builder does not appear to build an architecture compatible with supplied data. "
+const ERR_BUILDER = "Builder does not appear to build an architecture compatible with supplied data. "
```

Comment:
this line is too long as per Blue style guide. That is the reason for the line break you removed.

Reply:
@ablaom I did not really intend to remove; it's just that my formatter maybe has different settings and it did automatically. Will try to fix my formatter and this.

Reply:
For now, I fixed this instance.